_posts/2022-04-28-introducing-uswds-3-0.md

```diff
@@ -58,14 +58,14 @@ IE11 has fallen below our 2% support threshold as monitored by analytics.usa.gov
 
 [USWDS browser support (2% rule)](https://designsystem.digital.gov/documentation/developers/#browser-support) →
```

Comment:
```suggestion
[USWDS browser support (2% rule)]({{ site.baseurl }}/documentation/developers/#browser-support) →
```

Reply:
Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)